### DISCUSSION TITLE
Can we get touch input notes?

### DISCUSSION THREAD

**Discussion by tmsteph** [toeverything/AFFiNE#1645] [Ideas]
I'd love to see handwritten notes and images. 

From Mouse Click, Touch, or pen input. 

From ipad or touch laptop. 

https://roughjs.com/ or similar

**Comment by JackiePengp** [comment]
I've checked this one, would say it is really cool!! will pass through this message to our dev side team to see if it's okay to have those fabulous shapes included into our edgeless mode
